src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -21,28 +26,65 @@
  */
 public class AddCommandParser implements Parser<AddCommand> {
 
+    /**
+     * Throws a ParseException if the type is not specified correctly in the input arguments.
+     * If the type is specified correctly, it returns the type.
+     */
+    private static Type parseType(ArgumentMultimap argumentMultimap) throws ParseException {
+        Type type;
+        try {
+            type = ParserUtil.parseType(argumentMultimap.getValue(PREFIX_TYPE).get());
```

Comment:
Any benefit of declaring Type type outside the catch block?

Reply:
Rectified in commit "Improve coding style of AddCommandParser.java"